style.css

```diff
@@ -620,139 +620,211 @@ input:valid {
   justify-content: center;
 }
 
-#pop-outer {
-  height: 836px;
-  width: 375px;
-  width: max-content;
-  background-color: #fff;
-  border-radius: 16px;
+.mobile-cards {
+  display: grid;
   padding: 16px;
-  margin-left: 16px;
-  margin-top: 200px;
+  font-size: 16px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  border-radius: 16px;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transition: 2000ms ease-in-out;
+  transform: translate(-50%, -50%) scale(1);
+  height: 100%;
+  width: 100%;
+  z-index: 2;
+}
+
+.btn-close {
   position: absolute;
-  z-index: 9999;
-  display: none;
+  top: 0%;
+  right: 5%;
+  background: none;
+  margin-top: 42px !important;
+  margin-right: 17px;
+  border: 1px solid white;
+  color: white;
 }
 
-#popup-content {
-  font-size: 32px;
-  font-weight: 600;
-  line-height: 44px;
-  color: #172b4d;
-  margin-left: 2px;
-  margin-top: 12px;
-  width: 259px;
-  height: 135px;
+.card-image {
+  overflow: hidden;
+  box-sizing: border-box;
+  border-radius: 16px;
 }
 
-#popup-content h1 {
-  margin-left: 0;
+.card-image > img {
+  width: 100%;
+  min-height: 220px;
+  object-fit: cover;
+}
+
+.card-heading {
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 1.5em;
+  color: #172b4d;
   text-align: left;
 }
 
-#badges {
-  display: flex !important;
-  flex-direction: row;
-  gap: 8px;
+.lang-list {
   list-style: none;
   padding: 0;
-  justify-content: space-between;
+  display: flex;
+  justify-content: flex-start;
+  gap: 8px;
 }
 
-#badges li {
-  flex: 1;
+.li-mobile {
+  background: #ebf0ee;
+  color: #3a4a42;
   padding: 0.5em 0.75em 0.5em;
-  border-radius: 0.25em;
-}
-
-.badge {
   font-family: Inter, sans-serif;
   font-style: normal;
   font-weight: 600;
+  font-size: 0.75em;
+  line-height: 1em;
   letter-spacing: 0.03em;
-  font-size: 12px;
-  line-height: 16px;
-  background: #ebf0ee;
-  color: #3a4a42;
+  border-radius: 0.25em;
 }
 
-#js-p {
-  font-family: "Inter", sans-serif;
+.card-para {
   color: #344563;
-  font-weight: 400;
-  font-size: 16px;
-  line-height: 24px;
   text-align: left;
-  width: 311px;
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 1em;
+  line-height: 1.5em;
 }
 
-#js-p2 {
-  font-family: "Inter", sans-serif;
-  color: #344563;
-  font-weight: 400;
-  font-size: 16px;
-  line-height: 24px;
-  text-align: left;
-  width: 311px;
+.card-button {
+  display: flex;
+  gap: 12px;
+}
+
+.btn-live-source {
+  font-weight: 500;
+  font-size: 14px;
+  letter-spacing: 2px;
+  color: white;
+  text-decoration: none;
+}

```

Comment:
- [ ] We could use display: flex; to align the items to the center for a better UI and to be similar to the Figma template.